packages/loot-core/migrations/1694438752000_add_goal_targets.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE zero_budgets ADD column goal INTEGER DEFAULT 0;
+
+COMMIT;
```

Comment:
**💭 thought:** ‏When doing DB migrations - please think three steps ahead: will we need to do further migrations? Will we want to rename or alter this column in the near future?

Adding a new column won't cause any problems. Neither will creating a new table.

But deleting/renaming/altering existing columns/tables will be either VERY hard or possibly even impossible due to how server-sync works.

Reply:
I understand.  Unless someone steps up and is interested in developing the GUI, this PR is DOA in my mind.  There's no purpose to it if whichever DB migration introduced isn't utilized.  I don't have enough expertise to push this across the finish line alone and certainly don't want to destabilize syncing by moving in a direction that hasn't been fully developed.